docs/source/basic_usage.rst

```diff
@@ -1,66 +1,55 @@
 Basic Usage
 ===========
 
-Let's say you have an app **family** with a model like this:
+Let's say you have an app **shop** with a model like this:
 
-File: model.py ::
+File: **models.py** ::
 
-    class Kid(models.Model):
+    class Customer(models.Model):
         """
-        Model class Kid of family app
+        Model class Customer of shop app
         """
         happy = models.BooleanField()
         name = models.CharField(max_length=30)
         age = models.IntegerField()
         bio = models.TextField()
         wanted_games_qtd = models.BigIntegerField()
         birthday = models.DateField()
-        appointment = models.DateTimeField()
-
-To create a persisted instance, just call Mommy:
+        last_shopping = models.DateTimeField()
 
-File: test_model.py ::
+To create a persisted instance, just call Model Bakery:
 
-    # -*- coding:utf-8 -*-
+File: **test_models.py** ::
 
     #Core Django imports
     from django.test import TestCase
 
     #Third-party app imports
-    from model_mommy import mommy
-    from model_mommy.recipe import Recipe, foreign_key
+    from model_bakery import baker
 
-    # Relative imports of the 'app-name' package
-    from .models import Kid
+    from shop.models import Customer
 
-    class KidTestModel(TestCase):
+    class CustomerTestModel(TestCase):
```

Comment:
Hm... I tend to keep the docs strictly related to how Django's test case works because `pytest` is not a dependency for `model_bakery`. If we use the pytest code-style, people may think that model bakery requires pytest and this can influence their opinions on using or not the library. 

I've created #14 to improve our docs to also explain model_bakery alongside with pytest